apps/datadogcharts/datadog_charts.star

```diff
@@ -97,6 +107,15 @@ def main(config):
             ],
         )
         return render.Root(child = child)
+    elif query_response == None:
+        # The Datadog API sometimes returns > 50MB of data which causes a context deadline exceeded error
+        print("Error fetching data. Possibly too large of a response.")
+        print(query_response)
+        child = render.Marquee(
+            width = 48,
+            child = render.Text(content = "Error fetching data. Possibly too large of a response.", font = "6x13", color = "#fff"),
+        )
+        return render.Root(child = child)
```

Comment:
This was a bit hard to track down. Just kept getting "context deadline exceeded" with no other information. Payload size was too large from the DD API I think for some charts depending on the time range.